src/caustics/cosmology.py

```diff
@@ -83,6 +81,7 @@ def critical_density(self, z: Tensor, params: Optional["Packed"] = None) -> Tens
         -------
         Tensor
             The critical density at each redshift.
+            *Unit: solMass/Mpc^3*
```

Comment:
Should this be `solMass` or `SolMass` for consistency?

Reply:
Updated